flapjack/ext.py

```diff
@@ -105,6 +105,11 @@ def _generate_manifest():
                 old_args = build_options.get(args_key, [])
                 build_options[args_key] = old_args + args
 
+        config_env = config.module_extra_env(m['name'])
+        if config_env:
+            build_options.setdefault('env', {})
+            build_options['env'].update(config_env)
```

Comment:
This could be just `build_options.setdefault('env', config_env)` if `config.module_extra_env` returns en empty dict instead of None, as `{}` is falsey. But is good as it currently is.

Reply:
Thanks for the tip! I always find myself unable to remember the differences between truthy and falsey values in JS and Python.